homeassistant/components/neptune_apex/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Neptune Apex",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to Neptune Systems Apex Aquacontroller",
+        "description": "",
+        "data": {
+          "host": "IP Address",
+          "username": "Username",
```

Comment:
Please us as many common strings as possible. Here are the available common strings:
https://github.com/home-assistant/core/blob/1169ac568b9fdb87fbb30ead83fe441d5d3cef4f/homeassistant/strings.json#L20-L55

Here's an example:
https://github.com/home-assistant/core/blob/1169ac568b9fdb87fbb30ead83fe441d5d3cef4f/homeassistant/components/adguard/strings.json#L10

Reply:
Thanks, much better. Should I remove the en.json from translations, modify similarly, or leave alone?

Reply:
If you want to test your change it's ok to run the translations script to update the default English translation.